### ISSUE TITLE
local images

### ISSUE THREAD

**Issue by franfranfranfran** [sjbcodes/teacozy#2]
https://github.com/sjbcodes/teacozy/blob/master/index.html#L11
https://github.com/sjbcodes/teacozy/blob/master/resources/css/style.css#L58
etc...

nice job including your image resources with the project folder. make sure to use these images on the site itself by using relative file paths to link to them. more info can be found here - http://www.coffeecup.com/help/articles/absolute-vs-relative-pathslinks/

**Comment by sjbcodes** [comment]
Thanks – I had a strange thing happen here that I don't have an explanation for. I put in the online links to the images to get things happening quickly, and only realised that I'd forgotten to update them to absolute paths after committing to github. When I tried to change them and re-commit, the paths wouldn't update. It was right there in the code but not changing in the online version of the site. 

Long story short, I do understand the use of absolute paths and how to use them, and will be working with them right from the start next time to avoid this weird issue!
